Source/DafnyCore/assets/.resources/coverage.css

```diff
@@ -0,0 +1,90 @@
+body, td {
```

Comment:
Shouldn't this CLI UI be in DafnyDriver instead of DafnyCore?

Reply:
I think if I were to move these files to `DafnyDriver`, I would have to move the coverage reporting code there as well, so that they can be accessed from within the same assembly? My reason to put the assets in `DafnyCore` was that this mimics the set up for the Auditor.

Reply:
> I think if I were to move these files to DafnyDriver, I would have to move the coverage reporting code there as well, so that they can be accessed from within the same assembly?

Sounds good.

> My reason to put the assets in DafnyCore was that this mimics the set up for the Auditor.

Isn't the auditor code a little all over the place?

Reply:
I see, sorry, I think I misunderstood your original comment. I have now moved the `CoverageReportCommand.cs` and all the HTML parsing to `DafnyDriver`, with the tools for creating an in-memory coverage report remaining in `Dafny Core`.